drivers/bridge/bridge.go

```diff
@@ -135,7 +135,10 @@ func Init(dc driverapi.DriverCallback, config map[string]interface{}) error {
 		logrus.Debugf("Fail to initialize firewalld: %v, using raw iptables instead", err)
 	}
 	if err := iptables.RemoveExistingChain(DockerChain, iptables.Nat); err != nil {
-		logrus.Warnf("Failed to remove existing iptables entries in %s : %v", DockerChain, err)
+		logrus.Warnf("Failed to remove existing iptables entries in table %s chain %s : %v", iptables.Nat, DockerChain, err)
+	}
+	if err := iptables.RemoveExistingChain(DockerChain, iptables.Filter); err != nil {
+		logrus.Warnf("Failed to remove existing iptables entries in table %s chain %s : %v", iptables.Filter, DockerChain, err)
 	}
```

Comment:
Can you also please remove the isolation chain.
We can probably group the three cleanup calls in a function in `setup_ip_tables.go`